packages/ui/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@saleor/apps-ui",
+  "version": "1.0.0",
+  "devDependencies": {
+    "@saleor/macaw-ui": "0.8.0-pre.84",
```

Comment:
Please use Macaw version the same as apps, if we have 2 installed, it will fail. I think apps have it little older